sky/shell/platform/mojo/BUILD.gn

```diff
@@ -44,4 +49,11 @@ mojo_native_application("mojo") {
     "//third_party/icu",
     "//url",
   ]
+
+  if (is_android) {
+    ldflags = [
+      "-lGLESv2",
```

Comment:
answer: it does, and it's included if 'is_android' is true in the build. The mojo environment doesn't provide EGL entry points so this code if it runs will blow up. I don't think your code will ever call this, so the right thing to do is probably exclude it from the build, but that would require adding a distinction between the android-for-running-in-android and android-for-running-in-mojo builds